common/scripted_effects/wc_scourge_invasion_effects.txt

```diff
@@ -17,28 +17,21 @@ spawn_scourge_troops_effect = {
 			uses_supply = no
 			inheritable = yes
 			name = scourge_event_troops
-			levies = {
-				value = 2500
-			}
-			men_at_arms = {
-				type = light_footmen
-				stacks = 5
-			}
 			men_at_arms = {
-				type = bowmen
-				stacks = 4
+				type = ghoul
+				stacks = 8
 			}
 			men_at_arms = {
-				type = aerial_unit
-				stacks = 2
+				type = destroyer
+				stacks = 1
```

Comment:
I'd maybe use one regiment of frost wyrms instead of destroyers.

Reply:
> I'd maybe use one regiment of frost wyrms instead of destroyers.

I wanted some light aerial unit. Dragons are too rare and strong.

Reply:
What about the generic light aerial then (being gargoyles and stuff)? Or one generic and one cultural?

Reply:
Fair. I would rather use frost wyrms than because I like it solely cultural.

Reply:
Replaced in 1b26b5e3e01aad2efeb5459c53fc2dbe1e07af19.